container/make-manifest-list.py

```diff
@@ -156,8 +197,57 @@ def main():
             f'v{major}.{minor}.{micro}',
             f'v{major}.{minor}.{micro}-{datetime.today().strftime("%Y%m%d")}',
         ):
-        run_command_show_failure(
-          f'podman manifest push localhost/m {base}:{t}')
+        if args.dry_run:
+            print(f'skipping podman manifest push localhost/m {base}:{t}')
```

Comment:
what is `localhost/m`? I realize it's not being introduced as new, here, but a more descriptive name, and error message, would be good.

Reply:
localhost/m is just the scratch manifest for building locally before push.  It's noted at the top that it'll be used and destroyed if it exists, although I should be using it from the variable and not from a literal

Reply:
made sure to use the top-level variable LOCALMANIFEST.  I actually chose m because I thought it would be pretty unlikely to be the name of another local image.